Src/BootCamp.Chapter/Checks.cs

```diff
@@ -1,39 +1,36 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace BootCamp.Chapter
+﻿namespace BootCamp.Chapter
 {
     // This class is used to have a freedom of design, but with tests applied.
     public static class Checks
     {
         public static string FindMostPoorPerson(string[] peopleAndBalances)
         {
-            return "";
+            return "";  //BalanceStats.FindMostPoorPerson(peopleAndBalances); 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5053435)